lms/templates/emails/enroll_email_allowedmessage.txt

```diff
@@ -18,16 +18,16 @@ ${_("To access the course visit {course_about_url} and register for the course."
 % endif
 % else:
 
-${_("To finish your registration, please visit {site_name}/dashboard\n"
-        "You will need to login with Touchstone to access this page. If you "
-        "do not have a Touchstone account, please contact your instructor for "
-        "detailed directions.").format(
+${_("To access the course, you will first need to register for MITx at {site_name}"
```

Comment:
Should this be `Residential MITx` 

@NotoriousMKD

Reply:
yeah, that's a good idea.